xchange-lgo/src/main/java/org/knowm/xchange/lgo/LgoAdapters.java

```diff
@@ -47,9 +36,7 @@ public static ExchangeMetaData adaptMetadata(
       BigDecimal minAmount = product.getBase().getLimits().getMin();
       BigDecimal maxAmount = product.getBase().getLimits().getMax();
       Integer baseScale = currency.get(Currency.getInstance(product.getBase().getId())).getScale();
-      Integer priceScale =
-          currency.get(Currency.getInstance(product.getQuote().getId())).getScale();
-      BigDecimal increment = product.getQuote().getIncrement();
+      BigDecimal increment = product.getQuote().getIncrement().stripTrailingZeros();
```

Comment:
is this actually better then the previous implementation?

Reply:
Sorry, my PR description wasn't quite explicit. 
The issue here was that while the quote currency (USD) by itself has a scale of 4, in practice in a CurrencyPair it is limited to the price increment scale (without trailing 0).
This implementation avoid some confusions between the two, while retaining the capacity for LGO to change price increment value in the future. 

I also updated the `verifyOrder` in `LgoTradeService` to be more explicit (hopefully)

Reply:
ok